frontend/src/metabase/query_builder/components/expressions/Editor/HelpText/HelpText.tsx

```diff
@@ -0,0 +1,174 @@
+import cx from "classnames";
+import { Fragment, useCallback } from "react";
+import { t } from "ttag";
+
+import { useDocsUrl } from "metabase/common/hooks";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { Box, Flex, Icon, UnstyledButton } from "metabase/ui";
+import * as Lib from "metabase-lib";
+import { MBQL_CLAUSES } from "metabase-lib/v1/expressions/config";
+import {
+  getHelpDocsUrl,
+  getHelpText,
+} from "metabase-lib/v1/expressions/helper-text-strings";
+import type { HelpText } from "metabase-lib/v1/expressions/types";
+import type Metadata from "metabase-lib/v1/metadata/Metadata";
+
+import { HighlightExpression } from "../../HighlightExpression";
+
+import S from "./HelpText.module.css";
+
+function wrapPlaceholder(name: string) {
+  if (name === "…") {
+    return name;
+  }
+
+  return `⟨${name}⟩`;
+}
+
+export type HelpTextProps = {
+  open?: boolean;
+  onToggle?: () => void;
+  enclosingFunction?: {
+    name: string;
+    arg: {
+      index: number;
+    } | null;
+  } | null;
+  query: Lib.Query;
+  metadata: Metadata;
+  reportTimezone?: string;
+};
+
+function getDatabase(query: Lib.Query, metadata: Metadata) {
+  const databaseId = Lib.databaseID(query);
+  return metadata.database(databaseId);
+}
+
+export function HelpText({
```

Comment:
Can we get rid of the chevron-parens? They look very unusual.

![image](https://github.com/user-attachments/assets/4e78dfd1-6466-47f7-b703-7ee9630824f5)